Formula/cava.rb

```diff
@@ -0,0 +1,40 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake"
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "libtool"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+
+    system "./autogen.sh"
+    ENV.append "CFLAGS", "-g -O0"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! For Homebrew/homebrew-core
+    # this will need to be a test that verifies the functionality of the
+    # software. Run the test with `brew test cava`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    assert_match "cava", shell_output("#{bin}/cava -v").strip
```

Comment:
```suggestion
    assert_match "cava", shell_output("#{bin}/cava -v").strip
```
What happens if you try something more substantial? Can we attempt it anyway, try to get it to dump some logs and check that?

Reply:
cava has just two command line options:
  -v for printing the version
  -p for settings the path to the config file

It has an config option to send output, but this output is a raw printed audio output stream to stdout which.

Everything except "-v" is not reproducable...

Reply:
> -p for settings the path to the config file

We can check this too.

> It has an config option to send output, but this output is a raw printed audio output stream to stdout which.

Let's do this. We can at least check that doesn't exit with an error, which is what using `system` already does.

Reply:
> What do you mean by sort out?

See above. We can always dump all the output to `/dev/null` given that we're just checking that it doesn't error.